mmseg/models/data_preprocessor.py

```diff
@@ -48,18 +48,28 @@ class SegDataPreProcessor(BaseDataPreprocessor):
         rgb_to_bgr (bool): whether to convert image from RGB to RGB.
             Defaults to False.
         batch_augments (list[dict], optional): Batch-level augmentations
+        train_cfg (dict, optional): The padding size config in training, if
+            not specify, will use `size` and `size_divisor` params as default.
+            Defaults to None, only supports keys `size` or `size_divisor`.
+        test_cfg (dict, optional): The padding size config in testing, if not
+            specify, will use `size` and `size_divisor` params as default.
+            Defaults to None, only supports keys `size` or `size_divisor`.
```

Comment:
Please add an example in PR description for `train_cfg` and `test_cfg` as a padding config for input data.